src/app/exp-feature-page/flask.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I did drop it in. I had tried to use the file but couldn't get it to work. I left it there with the hope of using it.